src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/feedbackSubmissionForm.tag

```diff
@@ -23,13 +24,12 @@
 
   <t:statusMessage statusMessagesToUser="${data.statusMessagesToUser}" />
   <tsfse:feedbackSessionDetailsPanel feedbackSession="${data.bundle.feedbackSession}"/>
-
-  <c:forEach items="${data.questionsWithResponses}" var="questionWithResponses">
-    <tsfse:questionWithResponses isSessionOpenForSubmission="${data.sessionOpenForSubmission}"
-        isShowRealQuestionNumber="${data.showRealQuestionNumber}"
-        questionWithResponses="${questionWithResponses}"/>
-  </c:forEach>
-
+    <c:forEach items="${data.questionsWithResponses}" var="questionWithResponses">
+      <tsfse:questionWithResponses isSessionOpenForSubmission="${data.sessionOpenForSubmission}"
+          isShowRealQuestionNumber="${data.showRealQuestionNumber}"
+          questionWithResponses="${questionWithResponses}" isInstructor="${isInstructor}"
+          moderatedPersonEmail="${moderatedPersonEmail}"/>
+    </c:forEach>
```

Comment:
According to [the PR](https://github.com/TEAMMATES/teammates/pull/7762/files#diff-57370da4eb8173a385060d3da039ba65R27), the beginning line of this block has 2 spaces of indentation, as the line before it `<tsfse:feedbackSessionDetailsPanel feedbackSession="${data.bundle.feedbackSession}"/>` is self closing, and this block is therefore not a nested element of `tsfse:feedbackSessionDetailsPanel`.